jhipster-dependencies/pom.xml

```
@@ -51,32 +51,27 @@
         <commons-collections.version>4.4</commons-collections.version>
         <feign-reactor-bom.version>3.2.6</feign-reactor-bom.version>
         <guava.version>31.1-jre</guava.version>
-        <hazelcast.version>5.1.2</hazelcast.version>
+        <hazelcast.version>5.1.5</hazelcast.version>
         <hazelcast-hibernate5.version>2.2.1</hazelcast-hibernate5.version>
-        <infinispan.version>13.0.11.Final</infinispan.version>
+        <infinispan.version>14.0.3.Final</infinispan.version>
+        <!-- To pull in a newer version than Spring Boot with Nonnull -->
+        <jakarta-annotation.version>2.1.1</jakarta-annotation.version>
         <jjwt.version>0.11.5</jjwt.version>
-        <junit-platform-console.version>1.8.2</junit-platform-console.version>
+        <junit-platform-console.version>1.9.1</junit-platform-console.version>
         <logstash-logback-encoder.version>7.2</logstash-logback-encoder.version>
         <lz4-java.version>1.8.0</lz4-java.version>
         <mapstruct.version>1.5.3.Final</mapstruct.version>
         <mongock.version>5.2.1</mongock.version>
-        <mongodb.version>4.2.3</mongodb.version>
-        <mssql-jdbc.version>11.2.1.jre8</mssql-jdbc.version>
+        <mssql-jdbc.version>11.2.1.jre17</mssql-jdbc.version>
         <neo4j-migrations.version>1.15.0</neo4j-migrations.version>
-        <mysql-jdbc.version>8.0.30</mysql-jdbc.version>
+        <mysql-jdbc.version>8.0.31</mysql-jdbc.version>
         <oracle-jdbc.version>21.7.0.0</oracle-jdbc.version>
-        <problem-spring.version>0.27.0</problem-spring.version>
-        <problem-spring-web.version>${problem-spring.version}</problem-spring-web.version>
-        <problem-spring-webflux.version>${problem-spring.version}</problem-spring-webflux.version>
-        <prometheus-simpleclient.version>0.15.0</prometheus-simpleclient.version>
         <redisson.version>3.18.1</redisson.version>
-        <reflections.version>0.10.1</reflections.version>
-        <spring-cloud.version>2021.0.5</spring-cloud.version>
-        <spring-cloud-netflix.version>2.2.10.RELEASE</spring-cloud-netflix.version>
-        <spring-cloud-security.version>2.2.5.RELEASE</spring-cloud-security.version>
+        <reflections.version>0.10.2</reflections.version>
+        <spring-cloud.version>2022.0.0-RC2</spring-cloud.version>
```

```suggestion
        <spring-cloud.version>2022.0.0-RC3</spring-cloud.version>
```
https://spring.io/blog/2022/12/02/spring-cloud-2022-0-0-release-candidate-3-codename-kilburn-has-been-released